docs/CONFIG-VARS.md

```diff
@@ -28,13 +27,11 @@ Terraform input variables can be set in the following ways:
 
 ## Required Variables
 
-### Application
-
 | Name | Description | Type | Default | Notes |
 | :--- | ---: | ---: | ---: | ---: |
 | prefix | A prefix used in the name of all the Azure resources created by this script. | string | | The prefix string must start with a lowercase letter and contain only alphanumeric characters and dashes (-), but cannot end with a dash. |
 | location | The Azure Region to provision all resources in this script | string | "East US" | |
-| tags | Map of common tags to be placed on all Azure resources created by this script | map | { project_name = "sasviya4", environment = "dev" } | |
```

Comment:
Ditto again, "tags" I believe are required for some Azure resources

Reply:
Looks like "tags" can be left out of required section.

Reply:
So "tags" have are now in the "General" section.